rootfs/templates/wrapper-body.sh

```
@@ -0,0 +1,572 @@
+				DOCKER_ARGS+=(--env "$v=${!v}")
+			fi
+		done
+	fi
+
+	if [ "${WITH_DOCKER}" == "true" ]; then
+		# Bind-mount docker socket into container
+		# Should work on Linux and Mac.
+		# Note that the mounted /var/run/docker.sock is not a file or
+		# socket in the Mac host OS, it is in the dockerd VM.
+		# https://docs.docker.com/docker-for-mac/osxfs/#namespaces
+		echo "# Enabling docker support. Be sure you install a docker CLI binary${docker_install_prompt}."
```

Shellcheck (0.10.0)

[warning] docker_install_prompt is referenced but not assigned.

(SC2154)